subsquid/src/processorTypes.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  BatchBlock,
+  BatchContext,
+  BatchProcessorCallItem,
+  BatchProcessorEventItem,
+  BatchProcessorItem,
+  SubstrateBatchProcessor
+} from "@subsquid/substrate-processor";
+import { Store } from "@subsquid/typeorm-store";
+import { archive, chain, firstBlock } from "./config";
+
+console.log(`Chain ${chain()}`);
+console.log(`Archive ${archive()}`);
+
+export const processor = new SubstrateBatchProcessor()
+  .setDataSource({
+    chain: chain(),
+    archive: archive()
+  })
+  .setBlockRange({
+    from: firstBlock()
+  })
+  .addEvent("Pablo.PoolCreated", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Pablo.LiquidityAdded", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Pablo.LiquidityRemoved", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Pablo.Swapped", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Balances.Transfer", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Balances.Withdraw", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Balances.Deposit", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("BondedFinance.NewOffer", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("BondedFinance.NewBond", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("BondedFinance.OfferCancelled", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Vesting.VestingScheduleAdded", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addEvent("Vesting.Claimed", {
+    data: { event: { extrinsic: { signature: true, hash: true }, args: true } }
+  } as const)
+  .addCall("Pablo.add_liquidity", {
```

Comment:
The name of this event is unclear; What about naming it something like LiquidityAddedError? Same for Pablo.remove_liquidity, Pablo.swap

Reply:
This is autogenerated, and comes from the chain extrinsics (https://github.com/ComposableFi/composable/blob/3b22417b82f39127449fc80b315c29664c0d9283/code/parachain/frame/pablo/src/lib.rs#L387)